Header header
int32 ID
int32 turntable
int32 table_id
int32 time
